src/d3d11/d3d11_context_imm.cpp

```diff
@@ -3,6 +3,9 @@
 #include "d3d11_device.h"
 #include "d3d11_texture.h"
 
+constexpr static uint32_t MinFlushIntervalUs = 1251;
```

Comment:
Is there a reason the value was changed from 1250 to 1251?

Reply:
That wasn't intentional, I will change it back.